src/redux/constants.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  // sync
+  ADD_DUMMY: 'ADD_DUMMY',
```

Comment:
one of the reasons action types are constants is that javascript/babel can throw error when same constant name is used twice. With this notation its technically allowed to have same constant name, thats why  you will usually find in all examples notation with `export const ACTION` etc

Reply:
Good point. Fixed it! Thx.